functional_tests/scylla_operator/test_functional.py

```diff
@@ -849,3 +849,15 @@ def test_cloud_bundle_connectivity_scylla_bench(tester):
 
     # TODO: add verification that the output say it's using the cloud bundle
     #  (need to add that to log output in scylla-bench)
+
+
+@pytest.mark.required_operator("v1.8.0")
+@pytest.mark.requires_tls
+def test_cloud_bundle_connectivity_cqlsh(db_cluster: ScyllaPodCluster):
+
+    assert db_cluster.connection_bundle_file, "cloud bundle wasn't found"
```

Comment:
Not sure that we should fail the test in this case. It is kind of precondition.

Reply:
it can fail, and then we fallback to use the cqlsh direct, and it would work o.k. since the clear ports are still open.